### ISSUE TITLE
Publish latest version to npm?

### ISSUE THREAD

**Issue by mvattuone** [mapbox/leaflet-pip#22]
Hi there,

I recently upgraded to the 1.0.0 rc for Leaflet, and noticed some changes to leaflet-pip in the repo that do not seem to be available via the npm package. Would you be able to publish the latest version at some point? I'm currently adding it directly so I'm not blocked, but it would be nice to have as an installed npm module.

Thanks!

**Comment by gholol** [comment]
> but it would be nice to have as an installed npm module.

Just add the repo to your packages.json like this:

```
"dependencies": {
    "leaflet-pip": "https://github.com/mapbox/leaflet-pip#gh-pages",
}
```

**Comment by mvattuone** [comment]
Sure, but it'd be better to have semver support.

**Comment by tmcw** [comment]
Thanks for the heads-up - published 1.0.0

**Comment by mvattuone** [comment]
Thanks @tmcw !
